examples/common/async_asyncio_client.py

```diff
@@ -204,12 +201,12 @@ def run_with_no_loop():
     # Run with No loop
     log.debug("Running Async client")
     log.debug("------------------------------------------------------")
-    run_with_no_loop()
+    # run_with_no_loop()
 
     # Run with loop not yet started
     # run_with_not_running_loop()
 
     # Run with already running loop
-    # run_with_already_running_loop()
+    asyncio.run(run_with_already_running_loop())
```

Comment:
Just a general thought: 
We have to take a look closer on these Loop things.
My brain says, that there was something about multiple running loops shouldn't be made anymore...

Reply:
you sure can use multiple loops, e.g. one for the UI and one for collecting measurements, that is quite normal.